### DISCUSSION TITLE
Backend is offline (network error) after upgrade to 0.14.2

### DISCUSSION THREAD

**Discussion by elronn** [amidaware/tacticalrmm#1215] [Q&A]
Hi !

After upgrade my RMM to 0.14.2 version. The upgrade went well but i have this message when i try to login.

![image](https://user-images.githubusercontent.com/72046728/179716521-8fb59f59-e2bc-4227-b89b-82e8b1fc9ec7.png)

If i check the dev tools i have this message

Blocage d’une requête multiorigines (Cross-Origin Request) : la politique « Same Origin » ne permet pas de consulter la ressource distante située sur https://api.ajc-info.fr/checkcreds/. Raison : l’en-tête CORS « Access-Control-Allow-Origin » est manquant. Code d’état : 405.

Can i try to restore my last backup (in 0.13.4 version)

Thanks for your answers

**Comment by dinger1986** [comment]
You can restore your backup.

Can you also translate the error?

**Reply by elronn** [reply]
Ok i will try to restore my backup

the error in english

Blocking a Cross-Origin Request: The "Same Origin" policy does not allow you to view the remote resource on https://api.ajc-info.fr/checkcreds/. Reason: The CORS header "Access-Control-Allow-Origin" is missing. Status code: 405.

**Reply by dinger1986** [reply]
do you have any proxys or anything custom on it?

**Reply by silversword411** [reply]
400 errors are logged, look at the logs for more details.

Why/where did you get that API url from?

Gives me an error here.

Gives me a different error between your server and mine. mine returns proper API returns.

![Screenshot_20220719-083808.png](https://user-images.githubusercontent.com/13395348/179752050-7f0ecdf0-2674-4da7-bb8f-c394c7fa4eba.png)

Yours gives what looks like rmm. Error pages

**Reply by silversword411** [reply]
![Screenshot_20220719-083945.png](https://user-images.githubusercontent.com/13395348/179752350-80e68396-163a-416e-8d65-df44dc18f940.png)

**Comment by elronn** [comment]
nothing. A simple VPS

**Comment by silversword411** [comment]
You tried rebooting your server?

**Reply by elronn** [reply]
yes !  four time.  I restart the update process with --force flag. Everything is all right. All the services are started . I thing I will restore my previouce backup to see if something change

**Comment by elronn** [comment]
Ok ! I restore my previous backup and evrything is working and i have the new version. So, thank you for your help

**Comment by christopherquenneville** [comment]
I have the same problem after a fresh install.

Access to XMLHttpRequest at 'https://api.DOMAIN.TLD/checkcreds/' from origin 'https://mesh.DOMAIN.TLD' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Looking at the documentation, https://docs.tacticalrmm.com/troubleshooting/, there is a paragraph for CORS errors. My computer does ping the correct IP for all 3 subdomains. I am using cloudflare but I am not using a proxy. Reading this did not help me: https://developers.cloudflare.com/cache/about/cors/.

**Reply by silversword411** [reply]
Like the OP new VM, and retry the install logged (see docs). 

If you're still having problems create a new thread as this thread was resolved by reinstall and wasn't doing cloudflare something.

**Reply by christopherquenneville** [reply]
Fixed by reinstalling.
Thanks guys, you rock. Better response time than fortune 500 companies hahahaha.
